### DISCUSSION TITLE
Browser Sync

### DISCUSSION THREAD

**Discussion by christianthibault** [understrap/understrap-child#312] [Q&A]
Hi, 

I used to launch _gulp watch-bs_ (Browser Sync) while developing to avoid refreshing after each modification to Understrap Child. 

I downloaded the latest version, followed the instructions, modified the proxy value of _src/build/browser-sync.config.js_ to my MAMP local URL, launched _npm run watch_. CSS files are compiled upon saving _src/sass/theme/_child_theme.scss_, but it does not launch a synchronized browser window. Why?

Thank you, 

Christian Thibault

**Comment by 0dp** [comment]
Try `npm run watch-bs` since `npm run watch` will just look for file changes

**Reply by currentcreative** [reply]
Thank you...while probably obvious to experienced people, this should probably be in the README.md. And the README.md should probably not be tucked away in a hidden folder called .github.

Regardless, thanks to your comment, setting up a working child theme was WAY easier this time than the last site I set up a few months ago. I think the README file in that version was also hidden because I didn't see to run the copy assets command until after I'd already stumbled through manually copying and pasting stuff from Understrap to Understrap-Child to get things working.

**Reply by bacoords** [reply]
Thanks for pointing out the error in the readme. We have it ready to be updated in the next release.

**Comment by christianthibault** [comment]
Setting up the child theme is indeed easier than before. But the way the files are now compiled takes much longer than the gulp command. Often more than 2 seconds. Over a day of work, it adds up. 

Is there a way to speed up the process?

**Reply by bacoords** [reply]
This is a topic that comes up a lot, so we've been looking into to it. Part of the issue is that node-sass was a lot faster than dart sass, seems to be, but node-sass has been deprecated. We're trying out a few things.

**Comment by tribpapers** [comment]
I can't seem to find any documentation that speaks on how to use browser-sync with a site that is https:// .. I noticed that when I put the domain in the proxy it automatically adds the http:// prefix. Couldn't find anywhere to point to my local development certificates or keys.

**Reply by tribpapers** [reply]
Just stumbled upon this:
[https://gist.github.com/cwonrails/5c89f210f93343cea19c3faec348348c](https://gist.github.com/cwonrails/5c89f210f93343cea19c3faec348348c)

**Reply by bacoords** [reply]
Have you tried explicitly setting the https option? https://browsersync.io/docs/options/#option-https
